code-build.tf

```diff
@@ -77,7 +77,7 @@ resource "aws_codebuild_project" "tf_plan" {
 
   source {
     type      = "CODEPIPELINE"
-    buildspec = file("${path.module}/files/buildspec_tf_plan.yml")
+    buildspec = templatefile("${path.module}/files/buildspec_tf_plan.yml.tftpl", {TF_VERSION = "1.2.9", DIRECTORY = "${var.directory}", EXTRA_FILES = "${var.extra_build_artifacts}"})
```

Comment:
This seems to convert the `var.extra_build_artifacts` from set(string) into a string - how exactly it behaves, I don't know. But we want to give `templatefile()` function a (possibly empty) set of stings in `EXTRA_FILES`.